platform/engineering/design-docs/micro-frontend-architecture.md

```diff
@@ -0,0 +1,255 @@
+# Micro Front end Architecture
+
+**Author(s):** Chris Valarida (chris.valarida@adhocteam.us)  
+**Last Updated:** February XX, 2020  
+**Status:** **Draft** | In Review | Approved  
+**Approvers:** Andrew Gunsch, Rian Fowler, Wyatt Walter
+
+<!-- Should the approvers include anybody from DEPO? -->
```

Comment:
Dror has indicated he'd like to be a reviewer on all design docs once he gets here, at least initially while he ramps up and gets a handle on our processes